scripts/format_notebooks.py

```diff
@@ -207,9 +207,6 @@ def preprocess_cell(self, cell, resources, cell_index):
     if renumber_code:
         preprocessor_list.append(RenumberCodeCellPreprocessor)
```

Comment:
I think this one should go later too? Or are we sure that the renumber numbering is the same as the execution one?

Reply:
Running the entire notebook should run all of the cells in order - I thought that renumbering was to fix issues where cells had been run manually?

Reply:
Yeah, I think that's why it exists, but I think there might be a detail about code cells vs. text cells, where our manual numbering handles text cells differently to execution. However, maybe it does match in which case it doesn't matter.

I guess I'm just asking us to confirm that the numbering from `ExecutePreprocessor` is the same as from `RenumberCodeCellPreprocessor`.

Reply:
Our numbering changes only code cells (specifically the field `execution_count`, see https://ipython.org/ipython-doc/3/notebook/nbformat.html).  In my testing, the renumber behaviour of ascending integer numbering from 1 matches Jupyter's execution.

Reply:
Awesome; thanks for checking.